src/python/util/log.py

```diff
@@ -1,9 +1,21 @@
 import logging
 import os
 
-logger = logging.getLogger()
-logger.setLevel(logging.getLevelName(os.environ["LOG_LEVEL"]) if "LOG_LEVEL" in os.environ else logging.ERROR)
+def getChild(*args, **kwargs):
 
+    logger = logging.getLogger()
```

Comment:
move back up and should likely be `logging.getLogger('gak')` or `logging.getLogger(__name__)`